draft-ietf-tls-tls13.md

```diff
@@ -3264,17 +3273,18 @@ by an encrypted body, which itself contains a type and optional padding.
        } TLSInnerPlaintext;
 
        struct {
-           ContentType opaque_type = 23; /* application_data */
-           ProtocolVersion legacy_record_version = 0x0301; /* TLS v1.x */
-           uint16 length;
+           uint16 masked_length;
            opaque encrypted_record[length];
        } TLSCiphertext;
 
-content
-: The cleartext of TLSPlaintext.fragment.
 
-type
-: The content type of the record.
+:masked_length
+
+The length value with the high bit set. Records are always less than
+2^15 bytes, so this bit is unused. Having the high bit set allows
+receivers to easily distinguish TLS 1.3 encrypted packets from
+TLS 1.2 compatible framed plaintext packets (which start with the
+content type).
```

Comment:
Might want to point out that plaintext packets are always either alerts or handshake content types, which don't set the high bit.